### PR TITLE
adds underline text to standard reports landing page

### PR DESCRIPTION
**Pull request by DanielCouzens**
Please see this ticket https://github.com/epimorphics/standard-reports-ui/issues/114
Screenshot from 2024-08-12 14-40-33

### CODE REVIEW

File: app/views/report_design/_help.html.haml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 .o-help-link
   = link_to "https://www.gov.uk/guidance/price-paid-data-standard-reports#guidance-on-searching-the-data", class: "button button--secondary", alt: "Help and tips", title: "Help and tips" do
     %i.fa.fa-question-circle
-    help
-
+    help
```
Comment (bogdanadrianmarc) [inline app/views/report_design/_help.html.haml]: Github complaining about the lack of new line at the end of this file, otherwise it looks good to me 👍

**Review by bogdanadrianmarc — APPROVED** [review]

**Comment by jonrandahl** [comment]
## Please wait to merge!

Once the updated [Common Styles gem](https://github.com/epimorphics/lr_common_styles) has been released into the package repository, we will need to update the gemfile to use the "new" gem version otherwise Rails won't install the updated gem.

[line 83](https://github.com/epimorphics/standard-reports-ui/blob/ca36948e030130fcd07a11afbf995a913712171e/Gemfile#L83C3-L83C37) will need to read:

```ruby
gem 'lr_common_styles', '~> 1.9', '>= 1.9.4'
```

This is due to the way the [twiddle](https://depfu.com/blog/2016/12/14/get-to-know-your-twiddle-wakka) [wakka](https://alerner1st.medium.com/the-strange-case-of-the-twiddle-wakka-5a70a0f5a509) [works](https://bundler.io/guides/using_bundler_in_applications.html#adding-gems)

**Comment by DanielCouzens** [comment]
@jonrandahl Should we make a note/create a ticket to update the gem file when we are ready to go to pre-prod with the current changes and just keep updating the dev branch for this repo for now?

**Comment by jonrandahl** [comment]
> @jonrandahl Should we make a note/create a ticket to update the gem file when we are ready to go to pre-prod with the current changes and just keep updating the dev branch for this repo for now?

Yes and no... 

**Yes**, because the changes contained within this PR will be applied to the specific SR UI updates contained and therefore verifiable; however, the application will be missing the changes contained in the updated styles gem. Which leads us to the **No**, because this will require another release of each of the applications to include this new gem version.

If we choose to release often, which **_is_** a good thing, should we still increment the version cadence as a patch simply for the associated gem update?

cc: @bogdanadrianmarc

**Review by jonrandahl — APPROVED** [review]
Looks great @DanielCouzens! Nice work!

**Pull request merged**